plugins/chat/assets/javascripts/discourse/components/chat-channel.js

```diff
@@ -550,6 +552,8 @@ export default class ChatChannel extends Component {
 
     resetIdle();
 
+    await message.tryToEnsureMentionsLoaded();
```

Comment:
I don't think we need `tryTo` in the name, in essence any function is a try. What we are really doing is: ensuring mentions are loaded.